### PR TITLE
Deploy summary

### PR DESCRIPTION
**Pull request by kjlundsgaard**
Adds deploy info to releases overview
screen shot 2017-03-10 at 3 05 51 pm

### CODE REVIEW

File: src/sentry/static/sentry/app/views/releases/releaseOverview.jsx [hunk 1]
```diff
@@ -202,6 +221,28 @@ const ReleaseOverview = React.createClass({
               })
             }
           </ul>
+          <h6 className="nav-header m-b-1">{t('Deploys')}</h6>
+          <ul className="nav nav-stacked">
+            { !deploys.length ? this.renderEmpty() :
+              deploys.map((deploy) => {
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/views/releases/releaseOverview.jsx]: Note you don't need parens if its just a single arg.

```javascript
deploys.map(deploy => {
```

File: src/sentry/static/sentry/app/views/releases/releaseOverview.jsx [hunk 2]
```diff
@@ -202,6 +221,28 @@ const ReleaseOverview = React.createClass({
...
+                return (
+                  <li key={deploy.id}>
+                    <a href={`/${orgId}/${projectId}/?query=environment:${deploy.environment}+release:${version}`}>
```
Comment (benvinegar) [inline src/sentry/static/sentry/app/views/releases/releaseOverview.jsx]: You need to [encode these](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent) for use in a query string:

```javascript
${encodeURIComponent(deploy.environment)}
```

Also ... should `query` not also be quoted?

```javascript
<a href={`/${orgId}/${projectId}/query="environment:${encodeURIComponent(deploy.environment)}+release:${encodeURIComponent(version_}"`
```

**Review by benvinegar — CHANGES_REQUESTED** [review]
See comment re: `encodeURIComponent`

File: src/sentry/static/sentry/app/views/releases/releaseOverview.jsx (hunk 1 above)
Comment (kjlundsgaard) [inline src/sentry/static/sentry/app/views/releases/releaseOverview.jsx]: 👍

File: src/sentry/static/sentry/app/views/releases/releaseOverview.jsx (hunk 2 above)
Comment (kjlundsgaard) [inline src/sentry/static/sentry/app/views/releases/releaseOverview.jsx]: Made changes as discussed

**Review by benvinegar — APPROVED** [review]

**Pull request merged**